tests/node.rs

```diff
@@ -0,0 +1,51 @@
+mod test;
+use test::prelude::*;
+
+#[test]
+fn at_least_one_node_should_be_available() {
+    let contains_only_stackable_taints = |node: &Node| {
+        node.spec
+            .as_ref()
+            .and_then(|spec| spec.taints.as_ref())
+            .into_iter()
+            .flatten()
+            .all(|taint| {
+                taint.key == "kubernetes.io/arch"
+                    && taint.value == Some(String::from("stackable-linux"))
+            })
+    };
+
+    let client = TestKubeClient::new();
+    let mut nodes = client
+        .list_labeled::<Node>("kubernetes.io/arch=stackable-linux")
+        .items;
+    nodes.retain(contains_only_stackable_taints);
```

Comment:
I'm fine with looking at taints here, just a question: was it a conscious decision to not check for the ready state?

Reply:
Good point. The readiness is now checked.